Plugins/BTCPayServer.Plugins.Blink/BlinkLightningClient.cs

```diff
@@ -27,7 +27,8 @@ public class BlinkLightningClient : ILightningClient
 {
     private readonly string _apiKey;
     private readonly Uri _apiEndpoint;
-    public  string? WalletId { get; set; }
+    public string? WalletId { get; set; }
+    public string? WalletCurrency { get; set; }
```

Comment:
Just added this here but it needs to be populated from the `defaultWalletId` and then `walletById` queries

Reply:
Ok we changed the API so that you can now use a query like below to get the default wallet's currency without the round-trip

```gql
query GetNetworkAndDefaultWallet {
  globals {
    network
  }
  me {
    defaultAccount {
      defaultWallet{
        id
        currency
      }
    }
  }
}
```